dot/state/service.go

```diff
@@ -160,15 +192,15 @@ func (s *Service) Start() error {
 	// retrieve latest header
 	bestHash, err := LoadBestBlockHash(db)
 	if err != nil {
-		return fmt.Errorf("cannot get latest hash: %s", err)
+		return fmt.Errorf("failed to get best block hash: %s", err)
 	}
 
 	log.Trace("[state] start", "best block hash", fmt.Sprintf("0x%x", bestHash))
 
 	// create storage state
-	s.Storage, err = NewStorageState(db, trie.NewEmptyTrie())
+	s.Storage, err = NewStorageStateFromTrie(db, trie.NewEmptyTrie())
```

Comment:
why is this renamed? can you change it back to `NewStorageState`

Reply:
It matches the naming convention of the other helper methods and it is important information to know that the storage state is being create from the genesis trie, no?

I can change it back but I think this is an improvement.

Reply:
well, since there is not `NewStorageState` function, I think it's a bit confusing now. `FromTrie` implies there is a `NewStorageState` without trie

Reply:
Ok, no problem I will change it back if this is more confusing to you.